src/Symfony/Component/HttpClient/Retry/RetryStrategyInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpClient\Retry;
+
+use Symfony\Contracts\HttpClient\ResponseInterface;
+
+/**
+ * @author Jérémy Derussé <jeremy@derusse.com>
+ */
+interface RetryStrategyInterface
+{
+    public function isRetryable(int $retryCount, ResponseInterface $response, \Throwable $throwable = null): bool;
+
+    public function getWaitingTime(int $retryCount, ResponseInterface $response, \Throwable $throwable = null): int;
```

Comment:
this clearly need some phpdoc (at least to document which unit this is expected to return)